app/styles/prose.css

```
@@ -899,14 +899,50 @@ pre {
-	color: revert;
+
+/* Form controls inside blog content are otherwise left "unstyled" by preflight. */
+.prose :where(form, .demo)
+	:where(input, select, textarea, button):not([type='checkbox']):not(
+		[type='radio']
+	) {
+	font: inherit;
+	font-size: 1rem;
+	line-height: 1.25rem;
+
+	border: 1px solid var(--border-secondary);
```

Stylelint (17.3.0)

[error] Unexpected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)